default-config.json

```diff
@@ -934,34 +852,26 @@
           "type": "float",
           "description": "A double value describing the amount of volume reduction outside the cone defined by the coneOuterAngle attribute.",
           "default": 0.0
-        },
-        "debug": {
-          "description": "Show debug visuals.",
-          "type": "bool",
-          "default": false
         }
       }
     },
-    "zone-audio-source": {
```

Comment:
Sidenote, we may need to rethink the name of this...

Reply:
I think we agreed on leaving it until we merger a few components together so leaving like this for the moment.